src/main/java/ToDo.java

```diff
@@ -0,0 +1,28 @@
+public class ToDo extends Task {
+    public ToDo(String desc) {
+        super(desc);
+        taskType = TaskType.TODO;
+    }
+
+    @Override
+    public String toString() {
+        return "[" + taskType + "]" + super.toString();
+    }
+    
+    @Override
+    public String generateSaveFileData() {
+        return "T|" + (isDone ? "1" : "0") + "|" + desc;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Hi thank you for reviewing my PR! I had to implement an equals method because of my unit tests, which checks that the correct ToDo object is created when the input is "todo ...".